test/onerror.test.js

```diff
@@ -171,10 +171,12 @@ test.cb('should call middleware function on error in res stream app', t => {
   }
 })
 
+// fails when run with other tests. passes on it's own
 test.cb('should call middleware function on error in req stream app', t => {
```

Comment:
@anonrig @bojand I wonder if you could provide some assistance here. This test fails with the following error...

```
  Uncaught exception in test/onerror.test.js

  Error [ERR_STREAM_WRITE_AFTER_END]: write after end

  › node_modules/@grpc/grpc-js/src/call-stream.ts:765:26

  ✖ test/onerror.test.js exited with a non-zero exit code: 1
```

It passes when run on its own

Reply:
Resolved with colleague.